src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract void doWork();
+
+    public abstract void stopWork();
+}
+
```

Comment:
redundant empty line
```suggestion
```

Reply:
Unfortunately, I can't remove the empty row because the program isn't building
![Снимок экрана 2024-01-30 225411](https://github.com/mate-academy/jv-oop/assets/105318502/78e1e3eb-1841-4cc6-8691-ffd7041357c2)